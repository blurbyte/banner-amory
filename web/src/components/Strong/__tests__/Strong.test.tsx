import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import Strong from '../index';

test('renders correctly', () => {
  const component = renderer.create(<Strong />);
  expect(component.toJSON()).toMatchSnapshot();
});
