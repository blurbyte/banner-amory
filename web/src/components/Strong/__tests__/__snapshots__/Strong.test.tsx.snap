// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  font-weight: 600;
}

<strong
  className="c0"
/>
`;
